src/helpers/misc.ts

```diff
@@ -0,0 +1,61 @@
+import { FSXAApiErrors } from '../enums'
+
+type AvailableLocaleParams = {
+  navigationServiceURL: string
+  projectId: string
+  contentMode: string | ('preview' | 'release')
+}
+
+export const getAvailableLocales = async ({
+  navigationServiceURL,
+  projectId,
+  contentMode,
+}: AvailableLocaleParams): Promise<string[]> => {
+  try {
+    const queryParam = `${contentMode}.${projectId}`
+    const url = `${navigationServiceURL}?from=${queryParam}&until=${queryParam}`
+
+    //@TODO: use appropriate language headers
+    const caasApiResponse = await fetch(url, {
```

Comment:
it's not  caasApiResponse, but a response from the navigation service - the variable should be renamed.